Channels/Integration/01d_Amazon.md

```diff
@@ -0,0 +1,220 @@
+
+#### Procedure
+
+*Omni-Channel > Settings > Tab CONNECTIONS*
+
+![Amazon connection](../../Assets/Screenshots/Channels/Settings/Connections/Amazon/Connection.png "[Amazon connection]")
+
+1. Click the Amazon connection in the list of connections.   
+  The *Edit connection* view is displayed. By default, the *Credentials* tab is displayed.
+
+    ![Edit connection credentials](../../Assets/Screenshots/Channels/Settings/Connections/Amazon/Credentials.png "[Edit connection credentials]")
+
+2. Click the *Settings* tab.   
+  The *Settings* tab is displayed. By default, the *Offers* setting is displayed. In the right side bar, the *Apply from default* toggle is preselected and the *Update stock for 'Fulfillment by Amazon (FBA)' offers* is locked.  
+
+    > [Info] It is recommended to leave this settings unchanged, since Amazon manages stock for FBA offers.
+
+    ![Offers](../../Assets/Screenshots/Channels/Settings/Connections/Amazon/Settings_Offers.png "[Offers]")
+
+3. Click the *Order import* menu entry in the left side bar.    
+  The *Order import* setting is displayed in the right side bar.    
+
+    ![Order import](../../Assets/Screenshots/Channels/Settings/Connections/Amazon/Settings_OrderImport.png "[Order import]")
+
+4. Configure the following settings to filter the orders to be imported:
+
+  - In the *'Merchant Fulfilled Network' (MFN) orders* section: 
+    - Enable or disable the *Import 'Merchant Fulfilled Network' (MFN) orders* and *Import 'Prime by Merchant' orders* toggles to determine the MFN orders to be imported.  
+
+    - If desired, enter a date in the *Import MFN orders from* field to import orders from the entered date onwards. The valid date format is DD.MM.YYYY. 
+
+  - In the *'Fulfillment by Amazon' (AFN) orders* section:  
+    - Enable or disable the *Import 'Fulfillment by Amazon' (AFN) orders* toggle to import the AFN, also called FBA, orders. 
+  
+    - Enable or disable the *Mark 'Fulfillment by Amazon' orders as shipped during import and disable warehousing* to determine the further processing of AFN orders.  
+  
+    - Enable or disabled as necessary the *Is the VAT calculation services (VCS) enabled?* toggle. This feature is needed to determine whether the prices in the FBA reports are gross (VCS disabled) or net (VCS enabled).  
+
+    > [Info] When importing AFN orders, it is highly recommended to enable the VCS to makes sure that the gross/net prices are imported correctly from Amazon reports.
+
+    - If desired, enter a date in the *Import AFN/FBA orders from* field to import orders from the entered date onwards. The valid date format is DD.MM.YYYY. 
+
+  - In the *Addresses* section:
+    - Click the *Packstation address split* drop-down list and select the appropriate option. The following options are available:  
+      - **Customer number in address line**
+      - **Customer number in company** 
```

Comment:
```suggestion
4. Configure the following settings:

  + Enable the toggles of all offers you want to import in the *'Merchant Fulfilled Network' (MFN) orders* section. The following offers are available:
    - *Import 'Merchant Fulfilled Network' (MFN) orders*
    - *Import 'Prime by Merchant' orders* 

  + If desired, click the ![Calendar](../../Assets/Icons/Calendar.png "[Calendar]") (Calendar) button and select a date from which to import the orders. Alternatively, enter a date in the *Import MFN orders from* field. The valid date format is DD.MM.YYYY. 

  - In the *'Fulfillment by Amazon' (AFN) orders* section:  
    - Enable or disable the *Import 'Fulfillment by Amazon' (AFN) orders* toggle to import the AFN, also called FBA, orders. 
  
    - Enable or disable the *Mark 'Fulfillment by Amazon' orders as shipped during import and disable warehousing* to determine the further processing of AFN orders.  
  
    - Enable or disabled as necessary the *Is the VAT calculation services (VCS) enabled?* toggle. This feature is needed to determine whether the prices in the FBA reports are gross (VCS disabled) or net (VCS enabled).  

    > [Info] When importing AFN orders, it is highly recommended to enable the VCS to makes sure that the gross/net prices are imported correctly from Amazon reports.

    - If desired, enter a date in the *Import AFN/FBA orders from* field to import orders from the entered date onwards. The valid date format is DD.MM.YYYY. 

  - In the *Addresses* section:
    - Click the *Packstation address split* drop-down list and select the appropriate option. The following options are available:  
      - **Customer number in address line**
      - **Customer number in company** 
```

Reply:
![image](https://user-images.githubusercontent.com/99268392/211314106-7a1fc30e-3bb3-4742-8daa-371357016bd4.png)

Reply:
Wissenstransfer Video (ca. Minute 7:10): In den Amazon-Reports ist der alte Preis offenbar mal brutton mal netto ausgedrückt. Da bei FBA die Preise aus dem Bericht selber nicht herauslesen kann und davon aushängt, ob der VCS (UST-Berechnungsservice) aktiviert ist oder nicht, muss man die Angabe hier extra machen.